variables.tf

```diff
@@ -15,6 +15,12 @@ variable "api_servers_external" {
   default     = []
 }
 
+variable "api_servers_ips" {
+  description = "List of kube-apiserver IPs to include in the TLS certificate"
```

Comment:
Maybe `List of additional IPv4 addresses to be included in the kube-apiserver TLS certificate`?

Reply:
yes, done

Reply:
Actually, that could be `IPv4 CIRDs`, but `addresses` is fine as well.

Reply:
I would be surprised if TLS certificates could be issued for IP blocks.

Reply:
You're right, that doesn't make any sense :D